frontend/src/individual_add.tsx

```diff
@@ -449,7 +466,9 @@ export function IndividualAdd({
   const resetSibling = () => {
     const numberParts: string[] = individual?.number?.split("-");
     const sibling: Individual = {
-      number: numberParts ? numberParts[0] + "-" : null,
+      number: numberParts
+        ? numberParts[0] + "-" + numberParts[1][0] + numberParts[1][1]
```

Comment:
If I undertand recent discussions (https://nbisweden.slack.com/archives/C010DDHHZ4N/p1628502747044800?thread_ts=1628500448.031700&cid=C010DDHHZ4N), the this should be numberParts[0] dash the first three digits from numberParts[1]?